ui/pages/asset/components/token-buttons.tsx

```diff
@@ -74,6 +78,7 @@ const TokenButtons = ({
   const isBuyableChain = useSelector(getIsNativeTokenBuyable);
   const metaMetricsId = useSelector(getMetaMetricsId);
   const { openBuyCryptoInPdapp } = useRamps();
+  useBridging();
```

Comment:
On a similar note, I think we would need to code fence both or neither. Or we might call `useBridging` without importing it